src/events-shared.js

```diff
@@ -5,6 +5,13 @@ export const RESPONSE_PACKAGE_INFO_EVENT = overlayPrefix + 'RESPONSE_PACKAGE_INF
 export const READY_EVENT = overlayPrefix + 'READY_EVENT';
 
 export const dispatchEvent = (type, detail) => {
-  const event = new CustomEvent(type, { detail });
-  window.dispatchEvent(event);
+  window.postMessage({ type, detail }, '*');
```

Comment:
Is the `*` part of a test or purposely there?

Reply:
> A target origin can be specified using the `targetOrigin` member of options. If not provided, it defaults to "/". This default restricts the message to same-origin targets only.

I kept it as default, it is what we need.